### DISCUSSION TITLE
Error 500 after update

### DISCUSSION THREAD

**Discussion by JMcrafter26** [YOURLS/YOURLS#3360] [Q&A]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Self troubleshooting

- [X] I've read the [documentation](https://yourls.org/) and made sure to follow it.
- [X] I've read the [troubleshooting first steps](https://docs.yourls.org/guide/troubleshooting/first-steps.html) and [frequent issues](https://docs.yourls.org/guide/troubleshooting/frequent-issues.html) guides.

### Version

1.9.2

### Description

I just updated manually from 1.8.2 to 1.9.2 and after the update everything seems fine, expect of the main feature: I wasn't directed to the long url.
If I paste my short url in the Browser, it doesn't open and shows the error 500.

I searched online and tried many things, but it just keeps failing.
I also checked my php version and if there is any problem with my hoster, but there isn't.

### Expectation

That I will be redirected like normally

**Comment by dgw** [comment]
You have checked the box acknowledging that you followed the troubleshooting steps, but I don't see any logs here. There's nothing we can magically fix without knowing the error.

**Comment by cshenry13** [comment]
I'm getting this related to a 500 error after also upgrading to 1.9.2 with PHP 8.0 on Apache:

`[Fri Jun 24 09:36:18.898005 2022] [core:error] [pid 2651:tid 140123180607232] [client 47.227.85.252:60058] AH00124: Request exceeded the limit of 10 internal redirects due to probable configuration error. Use 'LimitInternalRecursion' to increase the limit if necessary. Use 'LogLevel debug' to get a backtrace., referer: https://something.com/admin/index.php`

Nothing else had changed until the update, but it's been broken since. After I removed some custom code from the htaccess file (now it's stock), I'm getting a 404 on every short link.

Maybe this is related?

**Reply by cshenry13** [reply]
Update: my hosting service changed some htaccess rules and got it working again. Still not sure what changed

**Comment by Pino4** [comment]
I have the same issue. Upgraded from 1.8.3 to 1.9.1 and got 500's on all urls and in the admin. PHP 7.4.3 and nginx.
Unfortunately I couldn't find anything of value in the logs.

**Comment by bwklol** [comment]
I have exactly same issue. Erro 500 in browser. No errrors log by apache2. It worked literally excellent before update. I consider rollback cause have backup. Must be something not correct with newest version.

Nevermind, found that plugin "[Case insensitive YOURLS](https://github.com/adigitalife/yourls-case-insensitive)" caused it, looks like plugin worked before update, after update it stopped working correctly.

**Reply by JMcrafter26** [reply]
I had the same Plugin and it caused this issue. But Here is an alternative: [https://github.com/seandrickson/YOURLS-Case-Insensitive](url) It is basically the same and it is working!

**Reply by ozh** [reply]
Pinging @adigitalife in case he is not aware of the problem

**Reply by adigitalife** [reply]
I've updated the README of my plugin to say it doesn't work with 1.9.1 or later and to also point to the plugin by @seandrickson.  Note, however, that my plugin is meant for Base 62 while Sean's one is meant for Base 36.

**Comment by aolko** [comment]
While updating to 1.9.2 for me it turned out to be composer complaining about php `7.3 < 7.4` and i **only** got that **after editing `admin/index.php` manually enabling error reporting**
